src/pages/Subscriptions/SubscriptionsAutoDeposit.vue

```diff
@@ -1,190 +1,188 @@
+									<div class="columns">
+										<span>
+											Optional donation to support Kiva
+										</span>
 									</div>
 
-									<div class="row">
-										<div class="columns">
-											<strong>Total/month</strong>
-										</div>
+									<div class="small-6 medium-4 columns">
+										<label
+											class="show-for-sr"
+											:class="{ 'error': $v.donation.$invalid }"
+											for="amount"
+										>
+											Donation
+										</label>
+										<kv-currency-input
+											class="text-input"
+											id="donation"
+											v-model="donation"
+										/>
+									</div>
+								</div>
+								<div class="row column align-middle">
+									<ul class="text-right validation-errors" v-if="$v.donation.$invalid">
+										<li v-if="!$v.donation.minValue || !$v.donation.maxValue">
+											Enter an amount of $0-$10,000
+										</li>
+									</ul>
+								</div>
 
-										<div class="small-6 medium-4 columns">
-											<strong
-												class="additional-left-pad-currency"
-											>{{ totalCombinedDeposit | numeral('$0,0.00') }}</strong>
-										</div>
+								<div class="row">
+									<div class="columns">
+										<strong>Total/month</strong>
 									</div>
-									<div class="row column">
-										<ul class="text-center validation-errors"
-											v-if="!$v.mgAmount.maxTotal || !$v.donation.maxTotal"
-										>
-											<li>
-												The maximum Auto Deposit total is $10,000.<br>
-												Please try again by entering in a smaller amount.
-											</li>
-										</ul>
+
+									<div class="small-6 medium-4 columns">
+										<strong
+											class="additional-left-pad-currency"
+										>{{ totalCombinedDeposit | numeral('$0,0.00') }}</strong>
 									</div>
 								</div>
+								<div class="row column">
+									<ul class="text-center validation-errors"
+										v-if="!$v.mgAmount.maxTotal || !$v.donation.maxTotal"
+									>
+										<li>
+											The maximum Auto Deposit total is $10,000.<br>
+											Please try again by entering in a smaller amount.
+										</li>
+									</ul>
+								</div>
 							</div>
-						</form>
-						<template slot="controls">
-							<kv-button
-								data-test="auto-deposit-save-button"
-								class="smaller button"
-								v-if="!isSaving"
-								@click.native="saveAutoDeposit"
-								:disabled="!isChanged || $v.$invalid"
-							>
-								Save
-							</kv-button>
-							<kv-button data-test="auto-deposit-save-button" class="smaller button" v-else>
-								Saving <kv-loading-spinner />
-							</kv-button>
-						</template>
-					</kv-lightbox>
-				</div>
-			</template>
-		</kv-settings-card>
-	</div>
+						</div>
+					</form>
+					<template slot="controls">
+						<kv-button
+							data-test="auto-deposit-save-button"
+							class="smaller button"
+							v-if="!isSaving"
+							@click.native="saveAutoDeposit"
```

Comment:
similar to comment above, I think this could be inside the form element, and be a `button type="submit"` with an `@submit="saveAutoDeposit"` on the form.